src/nidigital/system_tests/test_system_nidigital.py

```diff
@@ -210,12 +193,7 @@ def test_fetch_capture_waveform(multi_instrument_session):
 
     multi_instrument_session.pins['HighPins'].create_capture_waveform_parallel(waveform_name='capt_wfm')
 
-    multi_instrument_session.burst_pattern(
-        site_list='',
-        start_label='new_pattern',
-        select_digital_function=True,
-        wait_until_done=True,
-        timeout=5)
+    multi_instrument_session.burst_pattern(site_list='', start_label='new_pattern')
```

Comment:
This is gone, but you may want to review the rest of the tests to verify that you prefer passing appropriate time data types whenever applicable.